README.md

```diff
@@ -59,65 +55,78 @@ Execute `make` or `make help` to see the full list of project commands.
+deptrac-debug              Find files unassigned for Deptrac
+down                       Stop the docker hub
+e2e-tests                  Run end-to-end tests
+execute-load-tests-script  Execute single load test scenario.
+first-release              Generate changelog from a project's commit messages for the first release
+fix-perms                  Fix permissions of all var files
+infection                  Run mutation testing
+install                    Install vendors according to the current composer.lock file
+integration-tests          Run integration tests
+load-fixtures              Build the DB, control the schema validity, load fixtures and check the migration status
+load-tests                 Run load tests
+logs                       Show all logs
+new-logs                   Show live logs
+phpcsfixer                 A tool to automatically fix PHP Coding Standards issues
+phpinsights                Instant PHP quality checks and static analysis tool
+phpmetrics                 Get mathematical metrics
+phpunit-codecov            Get code coverage report
+psalm                      A static analysis tool for finding errors in PHP applications
+psalm-security             Psalm security analysis
+purge                      Purge cache and logs
+release                    Generate changelogs and release notes from a project's commit messages for the first release
+release-major              Generate changelogs and commit new major tag from a project's commit messages
+release-minor              Generate changelogs and commit new minor tag from a project's commit messages
+release-patch              Generate changelogs and commit new patch tag from a project's commit messages
+setup-test-db              Create database for testing purposes
+sh                         Log to the docker container
+smoke-load-tests           Run load tests with minimal load
+spike-load-tests           Run load tests with a spike of extreme load
+start                      Start docker
+stats                      Commits by the hour for the main author of this project stop                       Stop docker and the Symfony binary server
+stress-load-tests          Run load tests with high load
+unit-tests                 Run unit tests
+update                     Update vendors according to the current composer.json file
+up                         Start the docker hub (PHP, caddy)
+
+ ```
 
 ## Documentation
-Start reading at the [GitHub wiki](https://github.com/VilnaCRM-Org/php-service-template/wiki). If you're having trouble, head for [the troubleshooting guide](https://github.com/VilnaCRM-Org/php-service-template/wiki/Troubleshooting) as it's frequently updated.
+Start reading at the [GitHub wiki](https://github.com/VilnaCRM-Org/user-service/wiki). If you're having trouble, head for [the troubleshooting guide](https://github.com/VilnaCRM-Org/user-service/wiki/Troubleshooting) as it's frequently updated.
```

Comment:
We should have only links to the existing pages
https://github.com/VilnaCRM-Org/user-service/wiki/Troubleshooting
Please use this link instead
https://github.com/VilnaCRM-Org/user-service/wiki/Community-and-Support